### DISCUSSION TITLE
只能显示8个角色信息

### DISCUSSION THREAD

**Discussion by littleseven777** [Arondight/Adachi-BOT#805] [Q&A] [answered]
我查询[米游社]的时候能显示全部角色信息，但是群成员只能显示8个角色信息，这个在哪里调整啊。。。小白一枚

**Comment by Arondight** [comment, marked as answer]
目前来说，就是这个样子的

**Reply by littleseven777** [reply]
不是因为米游社的设置原因嘛

**Reply by Arondight** [reply]
官方的硬性设置，只有自己（的cookie）才能查看自己的全部角色

**Reply by littleseven777** [reply]
搜嘎

**Reply by littleseven777** [reply]
COOKIE总显示无效要怎么解决诶，都是一样的格式，有的账号的就可以用，有的就无效

**Reply by Arondight** [reply]
无效的就是无效，只能删除

**Reply by littleseven777** [reply]
为什么会出现无效的情况呢，检测语法也是对的，账号也能正常用

**Reply by Arondight** [reply]
买的号可能是没有初始化，老号可能是米游社被封禁，也有可能是没有用浏览器隐私模式提出的时候导致cookie过期，也有可能是米游社退出导致过期

**Reply by littleseven777** [reply]
好叭，蟹蟹
